ex03.ex

```diff
@@ -77,7 +91,17 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains([], value) do
```

Comment:
# 4

This is correct, but not ideal, as it falls into procedural code.

How about:

~~~ elixir
    def list_contains([], a), do: false
    def list_contains([h | tail], h), do: true
    def list_contains([h | tail], x), do: list_contains(tail, x)
~~~